iati/core/resources/test_data/202/valid_dateorder.xml

```diff
@@ -4,5 +4,9 @@
   <planned-disbursement type="1">
    <period-start iso-date="2014-01-01" /><!-- Dates are in the correct order -->
    <period-end iso-date="2014-12-31" />
+   <period-start iso-date="2017-01-01" /><!-- Dates are in the correct order -->
+   <period-end iso-date="NOW" />
+   <period-start iso-date="2014-01-01+5:00" /><!-- Dates are in the correct order but have timezone values -->
+   <period-end iso-date="2014-12-31+5:00" />
```

Comment:
Could also cover:

* Negative timezones
* Timezones that aren't on the hour
* `Z` rather than the `[\+-]\d?\d:\d\d` format